src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import { FilePlus, FolderPlus, Home, Share2, Files } from "lucide-react";
+
+export function Sidebar() {
+  return (
+    <div className="col-span-2 row-span-5 h-full bg-gray-200">
+      <div className="flex flex-col justify-between">
+        <div>
+          <div className="mb-2 ml-1 mr-1 mt-4">
+            <button
+              className="w-full rounded-full bg-blue-600 px-4 py-2 text-white hover:bg-blue-700"
+              aria-label="Upload files"
+            >
+              <div className="flex items-center justify-center">
+                <div className="min-w-6 min-h-6">
```

Comment:
You can add the classes directly to the icon instead of using a `div`. This applies to almost any icon in this file.